modules/installation-special-config-encrypt-disk.adoc

```diff
@@ -0,0 +1,155 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-customizing.adoc
+
+[id="installation-encrypting-disks_{context}"]
+= Encrypting disks during installation
+
+During {product-title} installation, you can enable disk encryption on all master and worker nodes.
+This feature:
+
+* Is available for IPI and bare metal {product-title} deployments
+* Is supported on Red Hat CoreOS (RHCOS) systems only
+* Sets up disk encryption during the manifest installation phase so all data written to disk, from first boot forward, is encrypted
+* Encrypts data on the root filesystem only (/dev/mapper/coreos-luks-root on /)
+* Requires no user intervention for providing passphrases
+* Uses AES-256-CBC encryption
+* Should be enabled for your cluster to be FIPS compliant.
+See link:https://docs.openshift.com/container-platform/4.3/installing/installing-fips.html[FIPS compliant {product-title} clusters]
+for details.
+
+There are two different supported encryption modes:
+
+* TPM v2: This is the preferred mode. TPM v2 stores passphrases in nvram.
+To implement TPM v2 disk encryption, create an Ignition config file as described below.
+
+* Tang: To use link:https://github.com/latchset/tang[Tang] to encrypt your cluster, you need to use a Tang server.
+link:https://github.com/latchset/clevis[Clevis] implements decryption on the client side.
+Tang encryption mode is only supported for bare metal installs.
+
+Follow one of the two procedures below to enable disk encryption for the nodes in your cluster.
+
+== Enabling TPM v2 disk encryption
+Use this procedure to enable TPM v2 mode disk encryption during {product-title} deployment.
+
+.Procedure
```

Comment:
There are multiple procedures in this file, so they could be modularized more and broken up into separate files if they're standalone procedures

Reply:
I put the two procedures in separate modules. The two modules on this topic are now: installation-special-config-encrypt-disk.adoc and installation-special-config-encrypt-disk-tang.adoc.